### PR TITLE
Using key down is enough

### PR DESCRIPTION
**Pull request by dzedro**
Look at the device in addon_products_yast2.pm & move to correct device with key 'down'

[addon_products_sle.pm](http://10.100.12.155/tests/3848#step/addon_products_sle/8)
[addon_products_yast2.pm](http://10.100.12.155/tests/3849#step/addon_products_yast2/11)

### CODE REVIEW

File: tests/installation/addon_products_sle.pm [hunk 1]
```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: rename sle's version of addon_products step
-# G-Maintainer: Stephan Kulow <coolo@suse.de>
+# Summary: rename sle's version of addon_products step
```
Comment (okurz) [inline tests/installation/addon_products_sle.pm]: Not a good summary

File: tests/installation/addon_products_sle.pm [hunk 2]
```diff
@@ -36,7 +36,6 @@ sub run() {
             wait_still_screen 3;
             send_key $cmd{next};
             assert_screen 'dvd-selector';
-            send_key_until_needlematch 'addon-dvd-list',         'tab',  10;    # jump into addon list
```
Comment (okurz) [inline tests/installation/addon_products_sle.pm]: Does it really harm to ensure that one is the right window?
Does it still work with SP1?

Comment (dzedro) [inline tests/installation/addon_products_sle.pm]: to ensure it is the right window there is 'addon-dvd-list' before
well this is not bif deal, just one assert less
yes it works also in SP1

File: tests/installation/addon_products_sle.pm (hunk 1 above)
Comment (dzedro) [inline tests/installation/addon_products_sle.pm]: updated

File: tests/installation/addon_products_sle.pm (hunk 2 above)
Comment (dzedro) [inline tests/installation/addon_products_sle.pm]: I meant 'dvd-selector' is there before :)

**Pull request merged**